app/controllers/api/application_controller.rb

```diff
@@ -3,4 +3,15 @@ class Api::ApplicationController < ActionController::API
   include GDS::SSO::ControllerMethods
 
   before_action :authenticate_user!
+  rescue_from ActiveRecord::RecordNotFound, with: :error_404
```

Comment:
I'm confused about the need for this: do we not return a `404` by default?

Reply:
Yup, I was trying to get around return a HTML formatted response. But realise I can set the default format to json in the routes and remove all this.